docs/rules/init-declarations.md

```diff
@@ -111,10 +115,14 @@ function foo() {
 }
 ```
 
-With `"ignoreForLoopInit"` enabled, the following pattern is not considered a problem.
+The `"never"` option ignores `const` variable initializations.
+
+### ignoreForLoopInit
+
+Examples of **correct** code for the `"never", {"ignoreForLoopInit": true}` options:
 
 ```js
-/*eslint init-declarations: [2, "never", { "ignoreForLoopInit": true }]*/
+/*eslint init-declarations: [2, "never", {"ignoreForLoopInit": true}]*/
```

Comment:
@IanVS Good catch! My quick search was wrong. In the rule files, eslint comments which have object literals: 122/187 = **65% with spaces** inside.

While we are at it, what about nested object literals? The 4 occurrences have a mixed message. Is operator-linebreak the pattern?

```js
/*eslint comma-style: [2, "first", {exceptions: {ArrayExpression: true, ObjectExpression: true} }]*/
/*eslint comma-style: [2, "first", {exceptions: {ArrayExpression: true, ObjectExpression: true} }]*/

/*eslint operator-linebreak: [2, "after", { "overrides": { "?": "ignore", ":": "ignore"} }]*/

/*eslint indent: [2, 2, {"VariableDeclarator": { "var": 2, "let": 2, "const": 3}}]*/
```

Can you confirm a related convention in the results: 139/187 = **74% with quotes** around property keys; here an example from the majority and the minority:

```js
/*eslint indent: [2, 2, {"VariableDeclarator": { "var": 2, "let": 2, "const": 3}}]*/
/*eslint one-var: [2, { var: "always", let: "never", const: "never" }]*/
```

In case you need some comic relief, my first ESLint-newbie reaction to block-spacing: 2 was “2 spaces, really?”